deepreg/model/network/build.py

```diff
@@ -20,39 +19,35 @@ def build_model(
     :param index_size: dataset size
     :param labeled: true if the label of moving/fixed images are provided
     :param batch_size: mini-batch size
-    :param model_config: model configuration, e.g. dictionary return from parser.yaml.load
-    :param loss_config: loss configuration, e.g. dictionary return from parser.yaml.load
+    :param train_config: train configuration
```

Comment:
Can you make the description consistent between files? "config for the model and loss" as it was referred to before.